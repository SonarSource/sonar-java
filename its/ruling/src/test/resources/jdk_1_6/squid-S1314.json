{
'jdk_1_6:java/awt/datatransfer/MimeType.java':[
346,
346,
],
'jdk_1_6:java/awt/datatransfer/MimeTypeParameterList.java':[
301,
301,
],
'jdk_1_6:java/lang/Terminator.java':[
35,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
206,
208,
211,
213,
],
}
