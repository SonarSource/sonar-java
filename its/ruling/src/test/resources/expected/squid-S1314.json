{
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
1138,
1160,
1217,
],
'project:com/sun/jmx/snmp/IPAcl/ASCII_CharStream.java':[
190,
],
'project:com/sun/jmx/snmp/IPAcl/ParserTokenManager.java':[
1120,
1329,
1481,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/AbstractDateTimeDV.java':[
63,
64,
],
'project:com/sun/security/auth/module/Crypt.java':[
255,
256,
257,
258,
314,
334,
],
'project:java/awt/datatransfer/MimeType.java':[
364,
364,
],
'project:java/awt/datatransfer/MimeTypeParameterList.java':[
318,
318,
],
'project:java/lang/Terminator.java':[
52,
],
'project:java/util/prefs/FileSystemPreferences.java':[
219,
221,
224,
226,
],
}
